File: PHP Word Examples/ConvertApiDemo/convertToBitmapInRequest.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\ConvertApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ConvertApi($configuration);

$inputFile = "D:/data/convertToBitmapInRequest.docx";
$image_format = "jpeg"; //Available image format: jpeg, bmp, png
$password = null;
$start_page = null;
$page_count = null;
$result = $apiInstance->convertToBitmapInRequest(
    $inputFile,
    $image_format,
    $password,
    $start_page,
    $page_count
);

File: Python Word Examples/ConvertApiDemo/convertToBitmapInRequest.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.convert_api.ConvertApi(configuration)

file_name = 'D:/input/ConvertToBitmapInRequest.docx'
image_format = 'jpeg' #Available values : jpeg, bmp, png
result = api.convert_to_bitmap_in_request(file_name, image_format)